e2e-tests/pitr/run

```diff
@@ -108,6 +117,15 @@ main() {
 	dest=$(sed 's,/,\\/,g' <<<$(kubectl get pxc-backup on-pitr-minio -o jsonpath='{.status.destination}'))
 	run_recovery_check_pitr "$cluster" "restore-on-pitr-minio" "on-pitr-minio" "select-4" "" "$dest" ""
 	echo "done latest type"
+	
+	desc 'check if we restore data from binlogs'
+	kubectl patch pxc pitr --type=merge -p '{"spec":{"backup":{"pitr":{"enabled":true}}}}'
+	sleep 20
+	run_backup "$cluster" "on-pitr-minio1"
+	write_more_data "$cluster"
+	dest=$(sed 's,/,\\/,g' <<<$(kubectl get pxc-backup on-pitr-minio1 -o jsonpath='{.status.destination}'))
+	sleep 80 # need to wait while collector catch new data
+	run_recovery_check_pitr "$cluster" "restore-on-pitr-minio1" "on-pitr-minio1" "select-5" "" "$dest" ""
```

Comment:
why we need similar check as in 109 line? can we add `run_backup ` and `write_more_data` before 108 line?